p1-staking-pool-dyn/src/lib.rs

```diff
@@ -174,16 +175,15 @@ impl Contract {
     pub fn close(&mut self) -> Promise {
         assert_one_yocto();
         let aid = env::predecessor_account_id();
-        let mut vault = self.get_vault_or_default(&aid);
```

Comment:
why change this?
please leave the name `get_vault_or_default`
that was the name Eugene suggested and I agree. 
`get_vault_or_default` implies not panicking, while `get_vault` does not

Reply:
This is not rename. `get_vault` is a new function which throws an exception when the vault doesn't exists.

`close` and `withdraw_crop` doesn't make sense for user who doesn't farm anything - that's why we want to fail early if user will like to game the contract.